tools/npz2rds/npz2rds.sh

```diff
@@ -80,7 +85,16 @@ if [ -f $outputfile ]; then
 fi
 
 ## Make tmp directory
-tmpdir=`mktemp -d`
+if [ -z "$tmpdir" ]
+then
+    tmpdir=`mktemp -d`
+else
```

Comment:
`else` isn't really necessary here since it wouldn't hurt to test the result of `mktemp` as well.